src/controller/BUILD.gn

```diff
@@ -101,6 +101,14 @@ static_library("controller") {
     }
   }
 
+  if (chip_device_config_enable_joint_fabric) {
```

Comment:
can this be a separate source set? That would make testing easier as one would not have to if-condution the tests.